protocol/agents.json

```diff
@@ -0,0 +1,118 @@
+{
+    "agents": [
+        {
+            "identifier": "ably-js",
+            "emitters": [
+                "ably-js"
+            ],
+            "type": "sdk"
+        },
+        {
+            "identifier": "node",
+            "emitters": [
+                "ably-js"
+            ]
+        },
```

Comment:
For consistency, can we give all entries a `type`, even if it's `type: "misc"` or something? Specifically here, I'd probably expect `node` to be `type: "runtime"`

Reply:
Is there a reason why it can't be optional? i.e. if it's not there (`undefined`) then it's implicitly "misc". 🤔

Reply:
I just feel like if you have an array of typed things, they should each have a type, and I'm a proponent of explicitness, especially in "common" things like this which is going to be shared between a variety of systems and users (SDKs, router, Grafana dashboards, Athena queries)

Reply:
Either way, all the current identifiers fit nicely into the `sdk|os|wrapper|runtime` tetrachotomy so no need for misc at the moment. fixed in e41c688.